# ==============================================================================
# FIXING COMMIT DIFF
# ==============================================================================
# File: gfx/thebes/gfxPlatform.h
# Commit: 425313809206
# Full Hash: 425313809206add668132f5653c0bba3a91e79d5
# Author: Matt Woodrow <mwoodrow@mozilla.com>
# Date: 2021-08-24 09:47:24
# Description:
#   Bug 1726562 - Remove option to disable software webrender for popups. r=jrmuizel
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D123159
# ==============================================================================

diff -r 60b0cb3f37fe -r 425313809206 gfx/thebes/gfxPlatform.h
--- a/gfx/thebes/gfxPlatform.h	Tue Aug 24 03:29:21 2021 +0000
+++ b/gfx/thebes/gfxPlatform.h	Tue Aug 24 04:06:34 2021 +0000
@@ -231,8 +231,6 @@
 
   static bool UseWebRender();
 
-  static bool DoesFissionForceWebRender();
-
   static bool UseRemoteCanvas();
 
   static bool IsBackendAccelerated(